tools/fq.wdl

```diff
@@ -8,24 +8,58 @@ version 1.1
 
 task fqlint {
     meta {
-        description: "This WDL task performs quality control on the input FASTQ pairs to ensure proper formatting."
+        description: "This WDL task performs quality control on the input FASTQs to ensure proper formatting."
     }
 
     parameter_meta {
-        read_one_fastq_gz: "Input FASTQ with read one"
-        read_two_fastq_gz: "Input FASTQ with read two"
+        read_one_fastq: "Input FASTQ with read one"
+        read_two_fastq: "Input FASTQ with read two"
```

Comment:
I would just note in the documentation here that gzipped or uncompressed inputs are supported.